tests/v2/conftest.py

```diff
@@ -1,124 +1,60 @@
+  yield afs_models.upload_model(model_path='unit_test_model',
+                            extra_evaluation={'extra_loss': 1.23},
+                            model_repository_name='test_model_repository',
+                            model_name='test_model')
 
-@pytest.fixture()
-def model_list_response():
-    return MockResponse(text="""
-    {
-          "pagination": {
-            "start": 0,
-            "limit": 10,
-            "total": 1
-          },
-          "resources": [
-            {
-              "uuid": "b8ed105e-c0b8-4b39-a453-0efd3259aafe",
-              "name": "test_model.h5",
-              "owner": "660939a7-edfb-4873-ba8a-1674ddb602d5",
-              "model_repository": "1a3a9596-8ee8-44b6-94f8-56ba70169300",
-              "evaluation_result": {
-                "accuracy": 0.1,
-                "loss": 0.2,
-                "metrics": 1
-              },
-              "parameters": {
-                "a": 1,
-                "b": 2
-              },
-              "tags": {
-                "machine": "01"
-              },
-              "created_at": "2019-01-15T10:07:54.966000+00:00"
-            }
-          ]
-    }""", status_code=200)
 
+@pytest.fixture(scope='function')
+def delete_model_respository(afs_models):
+  yield
+  afs_models.delete_model_repository(model_repository_name='test_model_repository')
 
-@pytest.fixture()
-def model_repository_list_response():
-    return MockResponse(text="""
-            {
-              "pagination": {
-                "start": 0,
-                "limit": 10,
-                "total": 1
-              },
-              "resources": [
-                {
-                  "uuid": "1a3a9596-8ee8-44b6-94f8-56ba70169300",
-                  "name": "test_model.h5",
-                  "owner": "660939a7-edfb-4873-ba8a-1674ddb602d5",
-                  "created_at": "2019-01-15T07:59:35.088000+00:00",
-                  "total_models": 0,
-                  "last_update": "2019-01-30T09:10:56.595000+00:00"
-                }
-              ]
-            }
-            """, status_code=200)
 
+@pytest.fixture(scope='function')
+def delete_mr_and_model(afs_models):
+  yield 
+  afs_models.delete_model(model_name='test_model', model_repository_name='test_model_repository')
+  afs_models.delete_model_repository(model_repository_name='test_model_repository')
 
-@pytest.fixture()
-def model_repository_response():
-    return MockResponse(text="""
-    {
-      "uuid": "11cc3faf-cd24-4d93-8f9a-09dd731f5397",
-      "name": "test_model.h5",
-      "owner": "75dca8a9-c6f7-4440-b8df-16989c0abca9",
-      "created_at": "2019-01-31T09:40:12.702000+00:00",
-      "total_models": 0,
-      "last_update": "2019-01-31T09:40:12.702000+00:00"
-    }""", status_code=200)
 
+@pytest.fixture(scope='function')
+def apm_node_env():
+  os.environ['PAI_DATA_DIR'] = """{
+      "type": "apm-firehose",
+      "data": {
+          "machineIdList": [3]
+      }
+  }"""
```

Comment:
Use `"""` carefully. It might be ruined by incorrect indentation. I suggest using dict and `json.dump()` to convert to string.

Reply:
It is nice suggestion. I will do it.